### PR TITLE
Updated readme

### PR DESCRIPTION
**Pull request by bhav09**
I was assigned the task [here](https://github.com/nvbn/thefuck/issues/1173#event-4445430636)

I have updated the readme file. I have updated it with the following:
1. **_Warp up link_**: to help the readers warp to the top at any point of time whilst reading/referring readme. This will save their time and prevent over scrolling across the readme. 
2.  **_Contents section_**: to help readers refer to a particular section of the readme without scrolling. 

The above-mentioned features do the following:
1. Improve readability.
2. Helps them keep a check of different sections present in the readme.
3. Looks more presentable.

Please do review this pull request and tell me if there are changes that I need to do.
Thanks for your consideration :)

### CODE REVIEW

**Comment by bhav09** [comment]
@scorphus, can you please review the PR? That would really be helpful.
Thanks in advance :)

File: README.md [hunk 1]
```diff
@@ -91,12 +91,27 @@ Reading package lists... Done
...
+9. [Developing](https://github.com/bhav09/thefuck#developing)
+10. [License](https://github.com/bhav09/thefuck#license-mit)
+
 ## Requirements
 
 - python (3.4+)
 - pip
 - python-dev
 
+##### [Warp to top](https://github.com/bhav09/thefuck#the-fuck----)
```
Comment (scorphus) [inline README.md:113]: Nit: how about point it to Contents and make it read "Back to Contents"?

Nit: And also maybe add a `0. TheFuck` or `0.. Top` — I'm not sure. Something that scrolls to the top of the readme.

**Review by scorphus — APPROVED** [review]
This is great! I love it! Thanks, @bhav09 👍 

Sorry for the late review. Please comment on the Nit's below :-)

File: README.md (hunk 1 above)
Comment (bhav09) [reply README.md:113]: Your idea is absolutely right. "Back to contents" would look better. I would update you with the changes in a while. 
Thanks, @scorphus :)

Comment (scorphus) [reply README.md]: Awesome! No hurry ;-)

Thank you!

**Comment by bhav09** [comment]
@scorphus, I have updated the markdown. Can you please have a look at it? Thank you :)

File: README.md [hunk 2]
```diff
@@ -156,6 +171,7 @@ To fix commands recursively until succeeding, use the `-r` option:
 ```bash
 fuck -r
 ```
+##### [Back to Contents](https://github.com/bhav09/thefuck#contents)
```
Comment (scorphus) [inline README.md:174]:
```suggestion

##### [Back to Contents](https://github.com/bhav09/thefuck#contents)
```

**Review by scorphus — CHANGES_REQUESTED** [review]
Just a tiny amendment

**Comment by bhav09** [comment]
@scorphus, done :)

**Comment by scorphus** [comment]
Thank you, @bhav09! 👍

**Comment by bhav09** [comment]
@scorphus, thank you very much for merging and helping me throughout :)

**Comment by scorphus** [comment]
You're welcome! Thank you for your contribution!

**Pull request merged**